packages/jaeger-ui/vite.config.mts

```diff
@@ -13,11 +13,11 @@
 // limitations under the License.
```

Comment:
* are the `*.mts` files included in our linter?
* instead of renaming the file, why not add `"type": "module"` to `./packages/jaeger-ui/package.json`?

Reply:
>are the *.mts files included in our linter?

no they arent.. forgot to check that. 

yeah you are right that would be better. making the change...